test/images/serve-hostname/serve_hostname.go

```diff
@@ -95,11 +95,11 @@ func main() {
 			log.Fatal(http.ListenAndServe(fmt.Sprintf(":%d", *port), nil))
 		}()
 	}
-
-	// Write to stdout after receiving SIGTERM and SIGINT to help with debugging kubernetes issue #21605
+	log.Printf("Serving on port %d.\n", *port)
 	signals := make(chan os.Signal)
-	signal.Notify(signals, syscall.SIGTERM, syscall.SIGINT)
+	signal.Notify(signals, syscall.SIGTERM)
 	sig := <-signals
-	// Keep behavior consistent with how the signal is handled by default (default is to panic)
-	log.Panicf("Terminating after receiving signal: %s.\n", sig)
+	log.Printf("Shutting down after receiving signal: %s.\n", sig)
+	log.Printf("Awaiting pod deletion.\n")
+	time.Sleep(60 * time.Second)
```

Comment:
@thockin @bgrant0607 @kubernetes/cncf-conformance-wg - this sleep in unnecessary and introduced regressions to the conformance suite in the wild.

Reply:
b/c this image is controlled by google, we need a googler to push a new image.

Reply:
xref - https://github.com/kubernetes/kubernetes/issues/71666

Reply:
This PR is over a year old...

Reply:
Let's consolidate discussion on the new issue